docs/development.rst

```diff
@@ -4,15 +4,18 @@
 Development
 ===========
 
-If you would like to add some features to ``tskit``, please read the
-following. If you think there is anything missing,
-please open an `issue <http://github.com/tskit-dev/tskit/issues>`_ or
+If you want to try out the development version of ``tskit``, or add features or
+documentation improvements, please read the following. If you think there is anything
+missing, please open an `issue <http://github.com/tskit-dev/tskit/issues>`_ or
 `pull request <http://github.com/tskit-dev/tskit/pulls>`_ on GitHub!
 
 **********
 Quickstart
 **********
 
+Create a local copy of the most recent (development) version
```

Comment:
This isn't really a heading --- needs to be short if it's going to work as a clickable link in the left sidebar.

Reply:
Yes, agree. How about:

```
Getting tskit (dev)
-------------------
```